### PR TITLE
[SW-262] Fix permissions layout

### PR DESCRIPTION
**Pull request by CedrikNikita**
- `PermissionsDetails`: on permission deletion, pressing back button was resulting in 404 
- margin for `BtnMain` was missed in #1601

### CODE REVIEW

File: src/popup/router/components/buttons/BtnMain.vue [hunk 1]
```diff
@@ -73,6 +73,7 @@ export default {
 
   @extend %face-sans-16-regular;
 
+  margin: 0 auto;
```
Comment (CedrikNikita) [inline src/popup/router/components/buttons/BtnMain.vue:76]: It was `margin: 8px auto;` not sure shall we have it 8 or 0.

File: src/popup/router/pages/PermissionsDetails.vue [hunk 2]
```diff
@@ -124,6 +124,7 @@ export default {
   font-size: 15px;
   text-align: left;
   color: variables.$color-white;
+  padding: 6px;
```
Comment (ifaouibadi) [inline src/popup/router/pages/PermissionsDetails.vue:127]: `padding: 16px;`
the common padding in settings screens are 16px

**Review by ifaouibadi — CHANGES_REQUESTED** [review]

File: src/popup/router/pages/PermissionsDetails.vue (hunk 2 above)
Comment (CedrikNikita) [reply src/popup/router/pages/PermissionsDetails.vue:127]: https://github.com/aeternity/superhero-wallet/blob/c6ebb4fcfe282e6ccccee15de9bf01860d21fa47/src/popup/pages/PermissionsSettings.vue#L50

**Comment by Liubov-crypto** [comment]
LGTM.

File: src/popup/router/components/buttons/BtnMain.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/router/components/buttons/BtnMain.vue:76]: In my opinion buttons should not have any margins on their own.
And also I really like the rule, that I red once, which says that the components should not position themselves (on their own). The parent should take care about placing the child elements in correct places. This is why I'm rather against adding x-axis margins as auto. The default behaviour of buttons in HTML is that they are inline-blocks so they align with the text.

File: src/popup/router/pages/PermissionsDetails.vue (hunk 2 above)
Comment (peronczyk) [reply src/popup/router/pages/PermissionsDetails.vue:127]: I suggest to use `--screen-padding-x` for x-axis paddings to maintain consistency within the pages.

**Review by peronczyk — CHANGES_REQUESTED** [review]

**Review by peronczyk — APPROVED** [review]

File: src/popup/router/pages/PermissionsDetails.vue (hunk 2 above)
Comment (peronczyk) [reply src/popup/router/pages/PermissionsDetails.vue]: Fixed

**Review by ifaouibadi — APPROVED** [review]

**Comment by peronczyk** [comment]
Closing the pull request as this was already done by @ifaouibadi .

**Pull request closed without merging**